tests/baselines/reference/multiLinePropertyAccessAndArrowFunctionIndent1.js

```diff
@@ -6,9 +6,8 @@ return this.edit(role)
 
 
 //// [multiLinePropertyAccessAndArrowFunctionIndent1.js]
-var _this = this;
 return this.edit(role)
     .then(function (role) {
-    return _this.roleService.add(role)
+    return this.roleService.add(role)

```

Comment:
Yes, it is intended. The `return` statement is an error so we don't do any further checks on the expression. It will never execute anyway.